exercise/serializer.py

```diff
@@ -50,17 +68,35 @@ def get_to_account(self, email):
         return email.to_account
 
 
+class EmailRevealTimeSerializer(serializers.HyperlinkedModelSerializer):
+
+    class Meta:
+        model = ExerciseEmailRevealTime
+        fields = ('reveal_time',)
```

Comment:
don't think we need this serializer right now and if we did we'd probably want to see the email and exercise ids too - I'd just remove it for now

Reply:
Removed serializer.